src/Cake.Tfs/PullRequest/GitClient/IGitClientFactory.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Cake.Tfs.PullRequest.GitClient
```

Comment:
I would suggest to move the factory to the `Cake.Tfs` namespace, as it might be used in the future for other stuff beside pull requests